azurerm/internal/services/web/resource_arm_function_slot.go

```diff
@@ -0,0 +1,657 @@
+package web
```

Comment:
I think the name of the file must be resource_arm_function_**app**_slot.go

Reply:
Thanks for your review!

Changed it! 🚀